packages/vscode-extension/test/extension/handlers.test.ts

```diff
@@ -592,6 +592,7 @@ describe("handlers", () => {
       });
 
       const res = await handlers.openConfigStateFile([]);
+      await fs.remove(tmpDir);
```

Comment:
minor: It's better to put this in afterEach() or use a library in case the test case fails.